network/panos/panos_admin.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+Ansible module to manage PaloAltoNetworks Firewall
+(c) 2016, techbizdev <techbizdev@paloaltonetworks.com>
+
+This file is part of Ansible
+
+Ansible is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Ansible is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+DOCUMENTATION = '''
+---
+module: panos_admin
+short_description: add or modify PAN-OS admins
+description:
```

Comment:
It would be helpful to provide some more descriptive text about what the purpose of the module is, how / why its used, etc.  Descriptions should always start with a capital letter and end with a full stop. Consistency always helps